### PR TITLE
clang_format is no executable in this package. 

### PR DESCRIPTION
**Pull request by jabbenseth**
<del>use clang-format-3.8 instead</del>

ask for clang

### CODE REVIEW

File: src/file_format.py [hunk 1]
```diff
@@ -42,7 +42,7 @@ def complete_path(text, state):
 readline.parse_and_bind('tab: complete')
 
 
-exec_policy = {'cpp': ' -name "*.h" -or -name "*.hpp" -or -name "*.cpp" | xargs rosrun ipa_code_refactoring clang_format -i -style=file',            # CPP
+exec_policy = {'cpp': ' -name "*.h" -or -name "*.hpp" -or -name "*.cpp" | xargs clang-format-3.8 -i -style=file',            # CPP
```
Comment (mgruhler) [inline src/file_format.py]: could this be made more generic? On later versions, there'll probably be anoter clang-format version...

Comment (jabbenseth) [inline src/file_format.py]: clang-format is rather uncool with this. They do not have a binary without the version suffix. We can ask for the path or the name to  the clang executable in the script and supply a default value for it.

**Comment by mgruhler** [comment]
CR: ok

**Comment by benmaidel** [comment]
CR: ok

**Pull request merged**